src/Raven.Server/Documents/DocumentsStorage.cs

```diff
@@ -1131,17 +1132,24 @@ private static void ThrowOnDocumentConflict(DocumentsOperationContext context, S
             var conflicts = GetConflictsFor(context, loweredKey);
             if (conflicts.Count > 0)
             {
-                var changeVectors = new List<ChangeVectorEntry[]>();
+                var conflictRecords = new List<GetConflictsResult.Conflict>();
                 foreach (var conflict in conflicts)
-                    changeVectors.Add(conflict.ChangeVector);
+                {
+                    conflictRecords.Add(new GetConflictsResult.Conflict
+                    {
+                        Doc = conflict.Doc,
+                        Key = conflict.LoweredKey,
```

Comment:
Why `LoweredKey`, and make sure that we don't hold to memory here from another context